rhobs/alerting/data_plane/prometheus.release_service_alerts.yaml

```diff
@@ -59,3 +59,27 @@ spec:
         alert_team_handle: <!subteam^S03SVBS426R>
         runbook_url: https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/release-service/release-processing-latency.md?ref_type=heads
         team: release
+
+    - alert: ReleaseServiceControllerManagerAvailability
+      expr: |
+        (
+          avg_over_time
+          (
+            (sum(kube_pod_container_status_ready{namespace="release-service", container="manager"}) by(namespace))
+            [1h:]
+          )
+          /
+          max_over_time(sum(kube_pod_container_status_ready{namespace="release-service", container="manager"}) by(namespace)[1d:])
```

Comment:
If we only keep a single namespace (`namespace="release-service"`), then I think we don't need to sum by namespace, as all data series will have the same namespace anyhow. i.e. I think we can drop the `by(namespace)`. No?

Reply:
I will check. I tried without it, for something broke. I might have mixed problems when I tried it.

Reply:
removed the `by(..)`